pkg/terraform/exec/plugins/vsphereprivate/resource_vsphereprivate_import_ova.go

```diff
@@ -255,12 +270,16 @@ func findImportOvaParams(client *vim25.Client, datacenter, cluster, datastore, n
 		}
 
 		if foundDatastore && foundNetwork {
+<<<<<<< HEAD
```

Comment:
oops ^

Reply:
> oops ^

Sorry for all the noise. That's what I get for trying to use fancy merge tools in vscode. Should be good now.

Reply:
no worries @willhaines, it happens to all of us ;-)